lib/ansible/modules/systemd.py

```diff
@@ -26,7 +26,8 @@
         description:
             - C(started)/C(stopped) are idempotent actions that will not run commands unless necessary.
               C(restarted) will always bounce the service. C(reloaded) will always reload.
-        choices: [ reloaded, restarted, started, stopped ]
+              C(try-restarted) will only restart the service if it is running.
```

Comment:
FTR I was going to name it `tried-restarting` once but never got to finish that PR: https://github.com/ansible/ansible/pull/40441/files#diff-1dbc7ca500073aa0f56b8d6d7de5b8bdR446-R447

I think that this form with both parts changed better suits the naming convention.

Reply:
`tried-restarting` for sure is more appropriate, I will change it.
But can't `try-restarted` be an alias of `tried-restarting` too?
I know that it's wrong and doesn't mean anything, but I'm saying this for the sake of similarity to other options (`started`, `stopped`, `restarted`, `reloaded`).
This would make some users more comfortable, but it's wrong. Is it appropriate or not?

Reply:
My personal opinion is to avoid all aliases if somehow possible, they usually only complicate things. Like in this case, the list of possible options would be increased by two choices, thus making the documentation more complex. Also, two choices have precisely the same meaning, which needs more documentation, and is harder to guess for users who are not familiar enough with the module (do they really do the same? or do they do slightly different things?).